daffodil-cli/src/main/scala/org/apache/daffodil/cli/Main.scala

```diff
@@ -1729,7 +1732,7 @@ class Main(
             ),
           )
 
-          if (fail == 0) ExitCode.Success else ExitCode.TestError
+          if (pass > 0 && fail == 0 && notfound == 0) ExitCode.Success else ExitCode.TestError
```

Comment:
Should we remove the `pass > 0` condition? I'm thinking of an an edge case where a tdml file has no tests defined and a user runs `daffodil test no_tests.tdml`. If we require `pass > 0` to be considered a success, then that command would exit with failure even though nothing technically failed. Maybe there's an argument that running no tests should be a failure? But using SBT as an example, running a test suite with no tests is considered a success. Maybe there are other counter examples?

Reply:
OK, agreed that pass > 0 can be removed.